src/mount/mastercomm.cc

```diff
@@ -806,7 +814,19 @@ int fs_connect(bool verbose) {
 			}
 #else
```

Comment:
I am not familiarized with this part. Consider adding commnets explaining why is this needed for Windows to mount the filesystem in META mode.

Reply:
Well, I'll try to explain a bit.

The previous if was "if (!gInitParams.meta)", so I means we are not mounting in META mode, to my understanding. Anyway, the lines in between the processor directives specify different behaviors for the Windows builds and the other builds. The Windows specific code here only makes consistent the use of client side uid/gid mappings and master side uid/gid mappings (this is the master exports.cfg file). So, we could have the case where the master defines some mapping and from the client side there is also some mapping defined, and this lines set it straight:
- it overrides the client side mapping if there is some master side mapping
- tells the user about the conection parameters (specifically the uid/gid final status)

For instance, this could be a conection parameters line (bold, the uid/gid final status is bold and italic):
>$ saunafscli -D H -H 192.168.195.48
**sfsmaster accepted connection with parameters: read-write,restricted_ip ; root mapped to 0:0 ; _users mapped to local IDs_**
The service saunafscli has been started.